<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVCalculator</class>
 <widget class="QWidget" name="IVCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IV Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="5">
       <widget class="QTextEdit" name="textEdit">
        <property name="toolTip">
         <string>Enter each entry on a new line with a space between each number</string>
        </property>
        <property name="placeholderText">
         <string>Level HP Atk Def SpA SpD Spe</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <layout class="QGridLayout" name="gridLayout_8" columnstretch="2,0,2,1,2">
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxAltForm"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelAltForm">
          <property name="text">
           <string>Altform</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelNature">
          <property name="text">
           <string>Nature</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelSpecies">
          <property name="text">
           <string>Pok√©mon</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButtonFindIVs">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Find IVs</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBoxNature"/>
        </item>
        <item row="2" column="0">
         <widget class="QComboBox" name="comboBoxPokemon"/>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxInformation">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBaseHP">
        <property name="text">
         <string>Base HP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelBaseHPValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBaseAtk">
        <property name="text">
         <string>Base Atk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelBaseAtkValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBaseDef">
        <property name="text">
         <string>Base Def</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelBaseDefValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelBaseSpA">
        <property name="text">
         <string>Base SpA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelBaseSpAValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelBaseSpD">
        <property name="text">
         <string>Base SpD</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelBaseSpDValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelBaseSpe">
        <property name="text">
         <string>Base Spe</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelBaseSpeValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxResults">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="4" column="0">
       <widget class="QLabel" name="labelSpDIV">
        <property name="text">
         <string>SpD</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelDefIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelSpAIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelHPIV">
        <property name="text">
         <string>HP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSpeIV">
        <property name="text">
         <string>Spe</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSpAIV">
        <property name="text">
         <string>SpA</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelSpeIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelHPIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelSpDIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAtkIV">
        <property name="text">
         <string>Atk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelAtkIVValue">
        <property name="text">
         <string>Invalid</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDefIV">
        <property name="text">
         <string>Def</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonSend">
        <property name="text">
         <string>Send to Seed Calculator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
